acados/ocp_nlp/ocp_nlp_cost_nls.h

```diff
@@ -63,6 +63,8 @@ void *ocp_nlp_cost_nls_dims_assign(void *config, void *raw_memory);
 void ocp_nlp_cost_nls_dims_initialize(void *config, void *dims, int nx, int nu, int ny, int ns, int nz);
 //
 void ocp_nlp_cost_nls_dims_set(void *config_, void *dims_, const char *field, int* value);
+//
+void ocp_nlp_cost_nls_dims_get(void *config_, void *dims_, const char *field, int* value);
```

Comment:
you need to do the same for `cost_external` too !

Reply:
The external cost doesn't contain data that can be set online, does it? Maybe just the parameters?

Reply:
Surely the parameter.

But in any case, generally speaking all instances of the same module must implement the same functions. This means that, in any case, there should be at least a function `ocp_nlp_cost_external_dims_get` which does nothing (or complains and exit).

As it is, if called though `config`, config->dims_get will give segfault for external cost...